README.md

```diff
@@ -42,14 +42,15 @@ GlobalTracer::set(new MyTracerImplementation());
 
 ### Creating a Span given an existing Request
 
-To start a new `Span`, you can use the `startActiveSpan` method.
+To start a new `Span`, you can use the `startSpan` method.
```

Comment:
I made this change because the example code just below actually uses `startSpan`. There is a sample that uses StartActiveSpan a little further down (and declares that it is the preferred method)